### ISSUE TITLE
Zero padding for max_pooling_layer

### ISSUE THREAD

**Issue by Abai** [nyanp/tiny-cnn#110]
I was comparing tiny_cnn to caffe and fiddling with your converter when I found out that tiny_cnn max pooling layer does not support padding. Do you think it would be a welcome addition to tiny_cnn to add this feature?

**Comment by nyanp** [comment]
@Abai
Padding in pooling layer is widely used and of course contribution is very welcome :)

**Comment by edgarriba** [comment]
@Wangyida Any idea how to deal with it in last version?

**Comment by Wangyida** [comment]
@Abai @edgarriba Yes, padding could be added cause message in proto could be used as cases for translation on binary file of caffe model, this might be a minor fix afterall as caffe also changed its own padding strategy via different versions such as the start pixel position of pooling index. It's not a fixed operation compared to the core function. I think it could be added later after core layers are all fixed this time.
